### ISSUE TITLE
Aggregations: handle stale aggregations after gappy sync

### ISSUE THREAD

**Issue by bwindels** [vector-im/element-web#10330]
Implement the solution [described in the MSC](https://github.com/matrix-org/matrix-doc/blob/9c90cc1db3391e072000f886aabed5a24f16ff7b/proposals/1849-aggregations.md#handling-limited-gappy-syncs) for dealing with stale aggregations after a gappy sync.

**Comment by bwindels** [comment]
Blocked on the MSC being finalized.

**Comment by manuroe** [comment]
Kamino cloned this issue to [vector-im/riotX-android](https://github.com/vector-im/riotX-android/issues/346)

**Comment by manuroe** [comment]
Kamino cloned this issue to [vector-im/riot-ios](https://github.com/vector-im/riot-ios/issues/2573)

**Comment by turt2live** [comment]
Closing in favour of https://github.com/vector-im/element-web/issues/9743
